src/command/list/human.rs

```diff
@@ -58,46 +108,215 @@ fn display_all(
     package_managers: &[PackageManager],
     packages: &[Package],
 ) -> String {
-    unimplemented!()
+    if runtimes.is_empty() {
+        NO_RUNTIME.to_string()
+    } else {
+        let runtime_versions: String = WRAPPER.fill(&format!(
+            "Node runtimes:\n{}",
+            format_runtime_list(runtimes)
+        ));
+        let package_manager_versions: String = WRAPPER.fill(&format!(
+            "Package managers:\n{}\n{}",
+            WRAPPER.fill("Yarn:"),
+            WRAPPER.fill(&format_package_manager_list(package_managers))
+        ));
+        let package_versions =
+            WRAPPER.fill(&format!("Packages:\n{}", format_package_list(packages)));
+        format!(
+            "⚡️ User toolchain:\n\n{}\n\n{}\n\n{}",
+            runtime_versions, package_manager_versions, package_versions
+        )
+    }
 }
 
-/// Format the output for `Toolchain::Node`.
-///
-/// Accepts only `runtimes` since this printer *should* be ignorant of all other
-/// types of `Toolchain` items.
+/// Format a set of `Toolchain::Node`s.
 fn display_node(runtimes: &[Node]) -> String {
     if runtimes.is_empty() {
         NO_RUNTIME.to_string()
     } else {
-        let width = text_width().unwrap_or(0);
-        let versions = Wrapper::new(width)
-            .initial_indent(INDENTATION)
-            .subsequent_indent(INDENTATION)
-            .fill(
-                &runtimes
-                    .iter()
-                    .map(|runtime| format!("v{}{}", runtime.version, runtime.source))
-                    .collect::<Vec<String>>()
-                    .join("\n"),
-            );
-
-        format!("⚡️ Node runtimes in your toolchain:\n\n{}", versions)
+        format!(
+            "⚡️ Node runtimes in your toolchain:\n\n{}",
+            format_runtime_list(&runtimes)
+        )
     }
 }
 
-/// Format the output for `Toolchain::PackageManager`.
+/// Format a set of `Toolchain::PackageManager`s.
 fn display_package_managers(package_managers: &[PackageManager]) -> String {
-    unimplemented!()
+    if package_managers.is_empty() {
+        String::from(
+            "⚡️ No <npm|Yarn> versions installed.
```

Comment:
Since the rest of this method is purely `Yarn` focused, can we update this to only say `Yarn`? Alternatively, we could make it fully language-agnostic, but it looks like that's a little bit more work (that can be tackled after this is merged in an update or follow-on to #694)